spec/models/spree/order_inventory_spec.rb

```diff
@@ -11,7 +11,7 @@ module Spree
 
       let(:bundle) { create(:product) }
 
-      before { bundle.parts.push [guitar, bass] }
+      before { bundle.master.parts.push([guitar, bass]); bundle.reload }
```

Comment:
Thoughts on breaking this from `{` block syntax to `do` `end` syntax? Not a big fan of the semicolon approach